packages/main/src/ColorPicker.ts

```diff
@@ -366,41 +366,43 @@ class ColorPicker extends UI5Element implements IFormInputElement {
 	}
 
 	_setMainColor(hueValue: number) {
-		if (hueValue <= 255) {
+		const hueValueMod = hueValue * 4.251;
```

Comment:
I know this magic number existed before, but this refactoring is an opportunity to set it to a constant.